Source/GmmLib/CMakeLists.txt

```diff
@@ -146,15 +146,24 @@ MESSAGE("SourceConfiguration:")
 MESSAGE("CommonDir: ${BS_DIR_COMMON}")
 MESSAGE("IncDir: ${BS_DIR_INC}")
 
+# If '-DGMM_KERNEL_5_1_MOCS_TABLE=TRUE' (default is FALSE) passed to cmake
+# configure command gmmlib will generate Fixed MOCS table depending on
+# usage requests (Gen11+). In this case on Linux user responsibility is to
+# make sure that same fixed MOCS table is programmed on KMD level.
+#
 # If '-DGMM_DYNAMIC_MOCS_TABLE=TRUE' (default is FALSE) passed to cmake
 # configure command gmmlib will generate MOCS table dynamically depending on
 # usage requests (Gen9+). In this case on Linux user responsibility is to
 # make sure that generated MOCS table is programmed on KMD level.
-if (GMM_DYNAMIC_MOCS_TABLE)
-    MESSAGE("MOCS table: dynamic")
+#
+if (GMM_KERNEL_5_1_MOCS_TABLE)
+    MESSAGE("MOCS table: Fixed Cross-OS Static MOCS Entries")
+    add_definitions(-DGMM_KERNEL_5_1_MOCS_TABLE)
+elseif (GMM_DYNAMIC_MOCS_TABLE)
+    MESSAGE("MOCS table: Dynamic MOCS entries")
     add_definitions(-DGMM_DYNAMIC_MOCS_TABLE)
 else()
-    MESSAGE("MOCS table: static")
+    MESSAGE("MOCS table: Limited 3 OTC MOCS Entries")
```

Comment:
don't use OTC, people outside Intel won't understand group name abbreviation. And I would not use "limited" either. Suggest: "MOCS table: 3 MOCS Entries matching Linux kernels <=5.1"